meta-refkit/recipes-devtools/nodejs/nodejs_4.5.0.bbappend

```diff
@@ -0,0 +1,17 @@
+# NodeJS needs Python for node-gyp which is a NPM module bundled with
+# GYP together. GYP is a build system generator written in Python that
+# stands for "Generate Your Projects" and is used for building Node
+# modules written in C. Since Refkit doesn't support on-target
```

Comment:
This isn't exactly true so let's be careful with the wording. Furthermore, even if refkit did not support this today, a refkit user might.

AFAICS this cripples node-gyp by keeping it installed (via node-npm) but simply drops dependencies to python. How about doing packaging such that npm and node-gyp are in their own packages.